src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-top: 3px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+}
+
+.nav__link {
+  margin-right: 20px;
+}
+
+.nav__link:last-child {
+  margin-right: 0;
+}
+
+.nav__item {
+  display: flex;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', serif;
```

Comment:
The font-family declaration includes 'Roboto' with a fallback to 'serif'. Since 'Roboto' is a sans-serif font, consider using a more appropriate fallback like 'sans-serif' to maintain visual consistency if 'Roboto' fails to load.